src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,481 @@
+  --pf-c-nav__list-tertiary-link--disabled--after--BackgroundColor:     transparent;
+
+
+  // Subnav links
+  --pf-c-nav__subnav--MarginTop:                                        var(--pf-global--spacer--sm);
+  --pf-c-nav__subnav--MaxHeight:                                        pf-size-prem(600px);
+  // automate this value based on previously set values
+  --pf-c-nav__subnav-item--PaddingLeft:                                 calc(var(--pf-global--spacer--xl) + var(--pf-global--spacer--md));
+
+  // List expand
+  // Set unique font size for dropdown caret based on design spec
+  --pf-c-nav__list-expand--FontSize:                                    pf-font-prem(20px);
+  --pf-c-nav__list-expand--Transition:                                  .1s ease-in-out;
+
+  // Nav section
+  --pf-c-nav__section--MarginTop:                                       var(--pf-global--spacer--xl);
+
+  // Nav section title
+  --pf-c-nav__section-title--PaddingTop:                                var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingRight:                              var(--pf-global--spacer--xl);
+  --pf-c-nav__section-title--PaddingBottom:                             var(--pf-global--spacer--sm);
+  --pf-c-nav__section-title--PaddingLeft:                               var(--pf-global--spacer--xl);
+  // Nav section title fonts
+  --pf-c-nav__section-title--FontSize:                                  var(--pf-global--FontSize--sm);
+  --pf-c-nav__section-title--Color:                                     var(--pf-global--Color--dark-200);
+  --pf-c-nav__section-title--FontWeight:                                var(--pf-global--FontWeight--semi-bold);
+  
+  /* stylelint-enable */
+
+  // Base styling
+  overflow-x: auto;
+
+  // Set width when nav is child of sidebar to prevent collapsing
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    .pf-l-page__sidebar & {
+      width: var(--pf-c-nav--Width);
+    }
+  }
+
+  // Hide scrollbars
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  // List default
+  &__list {
+
+    // Item
+    &-item {
+      position: relative;
+    }
+
+    // Link
+    &-link {
+      position: relative;
+      display: flex;
+      align-items: baseline;
+      padding:
+        var(--pf-c-nav__list-link--PaddingTop)
+        var(--pf-c-nav__list-link--PaddingRight)
+        var(--pf-c-nav__list-link--PaddingBottom)
+        var(--pf-c-nav__list-link--PaddingLeft);
+
+      font-weight: var(--pf-c-nav__list-link--FontWeight);
+      color: var(--pf-c-nav__list-link--Color);
+
+      // Adjust padding top for subsequent links
+      #{$namespace}__list-item:not(:first-child) & {
+        padding-top: var(--pf-c-nav__list-link--not-first-child--PaddingTop);
+      }
+
+      // States
+      &:hover,
+      &:focus,
+      &:active,
+      &.pf-m-active,
+      .pf-m-current > &,
+      &.pf-m-current {
```

Comment:
should these states be separated out so they have their own vars the customer can customize?

Reply:
Also, I think we talked about not using the modifier classes for states if the component in question would always have the native :state inherently. It hasn't made it into the code guidelines yet though.

Reply:
Updated